tools/tpm_manufacture.sh

```diff
@@ -75,7 +76,9 @@ openssl req -newkey rsa:2048 -nodes -keyout $CA_PRIV -x509 -days 365 \
 # Create and sign EK certificate
 openssl req -new -key $CA_PRIV -out $EK_CSR -config $EK_CONFIG
 openssl x509 -req -in $EK_CSR -CA $CA_CERT -CAkey $CA_PRIV -CAserial $CA_SRL \
-        -CAcreateserial -out $EK_CERT -outform der
+        -force_pubkey $EK_PUB \
```

Comment:
Can you move this around so that all CA parameters are grouped together?

Reply:
Fixed